deployments/docker-compose.yml

```diff
@@ -3,21 +3,35 @@ services:
   db:
     container_name: db
     build:
-      context: .
-      dockerfile: Dockerfile-db
+      context: ../
+      dockerfile: deployments/db/Dockerfile
     restart: always
     env_file:
       - ../.env
     ports:
       - "5555:5432"
     volumes:
-      - ./db:/docker-entrypoint-initdb.d/
+      - ./db/scripts:/docker-entrypoint-initdb.d/
       - database_volume:/var/lib/postgresql/data
-    healthcheck:
-      test: [ "CMD-SHELL", "pg_isready -U postgres" ]
-      interval: 5s
-      timeout: 5s
-      retries: 5
+
+  app:
+    container_name: app
+    build:
+      context: ../
+      dockerfile: deployments/app/Dockerfile
+    restart: always
+    command: /app/deployments/app/scripts/run.sh
+    volumes:
+        - ../:/app
```

Comment:
Why you use /app for mount if in Dockerfile we using /home/app/web as home dir for web app?